libc/hdr/unistd_overlay.h

```diff
@@ -0,0 +1,70 @@
+//===-- Including unistd.h in overlay mode
+//---------------------------------===//
```

Comment:
nit: fix formatting

Reply:
Oh, what about the CI complaining about the bad formatting then lol

Reply:
Fixed it, I think we'll just ignore the format CI fail then lol.